pages/common/mitmproxy.md

```diff
@@ -0,0 +1,11 @@
+# mitmproxy
+
+> An interactive man-in-the-middle HTTP proxy.
```

Comment:
I think you're right, but I think people tend not to think of a straightforward proxy as a "man-in-the-middle" because usually when you say "man-in-the-middle", you mean "man-in-the-middle attack". A simple proxy that just passes requests through blindly? Not really a MITM attack. But a proxy that strips out the SSL/TLS on HTTPS requests? That sounds more like an attack. But I dunno, that's just like, my opinion, man.

In any event, I think it's probably a good idea if our description spells out "man-in-the-middle" so people can figure out what the name means.